vpc/service/service.go

```diff
@@ -434,7 +434,7 @@ func (vpcService *vpcService) getTaskLoops() []taskLoop {
 		},
 		{
 			taskName:   "availability_zone",
-			itemLister: vpcService.getRegionAccounts,
+			itemLister: vpcService.getAllRegionAccounts,
```

Comment:
Is there any other worker that needs to do "All" region accounts instead of just "RegionAccounts" ? Like is the bug in any other woker?

Reply:
I looked through the other workers, and it doesn't seem so.

Reply:
A bunch of the other workers use region accounts, but they either are trunk eni specific or branch eni specific, and don't do both.